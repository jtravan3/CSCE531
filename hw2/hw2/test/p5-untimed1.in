x = 17
y = 6
z = 0-1
x
y
z
#4
#1
x = 14
#4
#1
#4*(#5+#6)
#7*(#5+#6)
z = 2*z - 1
y = #5 + 1
x = #10 - #2
#13
