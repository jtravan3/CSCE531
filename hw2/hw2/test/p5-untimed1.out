Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 17
   2: 6
   3: -1
   4: 17
   5: 6
   6: -1
   7: 17
   8: 17
   9: 14
  10: 14
  11: 17
  12: 70
  13: 70
  14: -3
  15: 7
  16: 8
  17: 32
  18: 
